packages/ai-client/tests/mcp-app-bridge.test.ts

```
@@ -0,0 +1,230 @@
+import { describe, expect, it, vi, beforeEach, afterEach } from 'vitest'
```

ESLint

[error] Member 'beforeEach' of the import declaration should be sorted alphabetically.

(sort-imports)